### DISCUSSION TITLE
Please make GitHub USEABLE!!!

### DISCUSSION THREAD

**Discussion by asonenblick-gpsw** [community/community#12054] [Other Feature Feedback, Questions, & Ideas]
GitHub's VERY LIMITED display of branch names is preventing me from being able to identify the the one I need.  PLEASE WIDEN THE PICKER!  Such a simple thing to do!

![image](https://user-images.githubusercontent.com/10039028/155576232-358f4f89-52ef-40ae-b61f-75d4ce7690dd.png)

**Comment by daniloab** [comment]
A workaround to help you (until having a specific fix or improvement):

try to inspect the element to get the whole name from branch
![Screen Shot 2022-02-24 at 16 24 38](https://user-images.githubusercontent.com/19939822/155593102-e3792b0d-680e-4b29-82ad-1e18032f2d83.png)

**Reply by queenofcorgis** [reply]
Thanks for sharing this workaround @daniloab :octocat:
